packages/runtime/test-runtime-utils/src/assertionShortCodesMap.ts

```diff
@@ -8,693 +8,698 @@
 // Auto-generated by policy-check in @fluidframework/build-tools.
 
 export const shortCodeMap = {
```

Comment:
Not sure if this addition is right - this is the result of running flub's assertion tagging tool on the repo. I'm surprised it added every assertion message in the repo here

Reply:
I'd expect it, the infrastructure to generate this file during assert tagging came into being once LTS wasn't the active branch. However, as I mentioned in another comment, I'd vote for not running assert tagging if we can avoid it. Not the end of the world if we have to, though (and this file is purely a development-time helper so it's fine if it ends up here).